frontend/src/components/RotateCredentials.vue

```diff
@@ -67,7 +68,10 @@ export default {
     },
     type: {
       type: String,
-      required: false
+      required: true,
```

Comment:
Define the property `type` in the mixin because the mixin already uses this property.